agent/tool-scripts/datalog/mock-lib/ansible/inventory/manager/__init__.py

```diff
@@ -0,0 +1,9 @@
+class InventoryManager:
+    def __init__(self, loader, sources):
+        self.loader = loader
+        self.sources = sources
+
+    def get_groups_dict(self):
+        masters = ["masterA.example.com"]
+        nodes = ["pprofA.example.com"]
+        return {"masters": masters, "pprof": nodes}
```

Comment:
That's fine, although I would have just done `return {"masters": ["masterA.example.com"], "pprof": ["pprofA.example.com"]}`, which would clarify that these are hardcoded literals and make it just a bit easier to read.

Reply:
Me, too...but I figured we would rather spend time on other stuff.  😏